kernel/kernel-core/src/main/java/io/mosip/kernel/core/util/CryptoUtil.java

```diff
@@ -90,7 +91,7 @@ public static String encodeBase64String(byte[] data) {
 	 * @return decoded data
 	 */
 	public static byte[] decodeBase64(String data) {
-		return Base64.decodeBase64(data);
+		return Base64.getUrlDecoder().decode(data.getBytes());

```

Comment:
These changes looks fine, but you have to change the version 1.1.5.5 in all the pom.xml files also

Reply:
done please check @vishwa-vyom